topdown/http_test.go

```diff
@@ -972,6 +972,44 @@ func TestHTTPSClient(t *testing.T) {
 		runTopDownTestCase(t, data, "http.send", rule, resultObj.String())
 	})
 
+	t.Run("HTTPS Get with Inline Cert", func(t *testing.T) {
```

Comment:
It might be good to add some test cases for when an invalid certificate string is provided, and since this supports aggregating sources for them maybe even like a test case for an inline cert and file cert w/ an error on one of them (to make sure it doesn't just make the request with only partial certs).

Maybe one test for just specifying the `tls_ca_cert` without client cert/key too. In the past we had them pretty tightly coupled and I want to make sure we have tests to ensure they are not tied together. The implementation does the right thing but it couldn't hurt to add more testing around this behavior with the new options.

Reply:
Sounds good; I'll look into adding those test cases.

Reply:
Added the extra test cases.